dd-sdk-android/src/main/kotlin/com/datadog/android/core/internal/system/BroadcastReceiverSystemInfoProvider.kt

```diff
@@ -91,10 +98,27 @@ internal class BroadcastReceiverSystemInfoProvider(
             val powerManager = context.getSystemService(Context.POWER_SERVICE) as? PowerManager
             val powerSaveMode = powerManager?.isPowerSaveMode ?: false
             systemInfo = systemInfo.copy(
-                powerSaveMode = powerSaveMode
+                powerSaveMode = powerSaveMode,
+                hasEnoughPower = !powerSaveMode
```

Comment:
Yes my mistake there, should only be setting `hasEnoughPower=false` in case `powerSaveMode=true` otherwise it should keep the old value. In any case I am going to change it again and follow the last suggestion.